deploy/docker/notebook/base/base.yaml

```diff
@@ -429,67 +429,56 @@ addfiles:
           
           #### Hook Class ####
 
-          class hookVariables():
-              POLUS_NOTEBOOKS_HUB_API = os.environ["POLUS_NOTEBOOKS_HUB_API"]
-              work_path = '/home/jovyan/work/'
-              shared_path = '/home/jovyan/shared/notebooks'
-
           class HookFileContentsManager(FileContentsManager):
-              """Local spawner that runs single-user servers as the same user as the Hub itself.
-              Overrides user-specific env setup with no-ops.
+              """ Jupyter Server Hook Class for database synchronization with Notebooks Hub
               """
 
+              POLUS_NOTEBOOKS_HUB_API = os.environ["POLUS_NOTEBOOKS_HUB_API"]
+              work_path = '/home/jovyan/work/'
+              shared_path = '/home/jovyan/shared/notebooks'
+              
               def is_inside_path(self, path_to_test, path):
                   return os.path.samefile(os.path.commonpath([path, path_to_test]), path)
               
               def script_post_delete(self, os_path):
 
-                  hookVar = hookVariables()
-
                   os_path = str(Path().resolve()) + os_path
                   
-                  if not (self.is_inside_path(os_path, hookVar.work_path) or self.is_inside_path(os_path, hookVar.shared_path)):
+                  if not (self.is_inside_path(os_path, self.work_path) or self.is_inside_path(os_path, self.shared_path)):
                       return
                   
                   try:
-                      r = requests.post(hookVar.POLUS_NOTEBOOKS_HUB_API+"/file-data/delete-file", json={"user": os.getenv("JUPYTERHUB_USER"), "filePath": os_path}, timeout=1)
+                      r = requests.post(self.POLUS_NOTEBOOKS_HUB_API+"/file-data/delete-file", json={"user": os.getenv("JUPYTERHUB_USER"), "filePath": os_path}, timeout=1)
```

Comment:
Please, use `urllib.parse.urljoin` to join the parts of URL: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin